types/simple-get/index.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for simple-get 4.0
+// Project: https://github.com/feross/simple-get
+// Definitions by: Konrad Perlicki <https://github.com/KonradPerlicki>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import { ClientRequest, IncomingMessage, RequestOptions } from 'http';
+import { URL } from 'url';
+import * as FormData from 'form-data';
+import { ReadStream } from 'fs';
+
+interface SimpleGetOptions extends RequestOptions {
+    url: string;
+    /**
+     * Default value: ```10```
+     */
+    maxRedirects?: number;
+    body?: string | {
+        [key: string]: unknown;
+    } | FormData | ReadStream;
+    form?: {
+        [key: string]: unknown;
+    };
+    followRedirects?: boolean;
+    /**
+     * Serialize/deserialize request and response with JSON
+     */
+    json?: boolean;
+}
+
+export = simpleGet;
+
+type Callback = (err: Error | null, res?: IncomingMessage) => void | never;
+
+declare function simpleGet(options: string | SimpleGetOptions | URL, cb: Callback): ClientRequest | never;
```

Comment:
```ts
declare function simpleGet(options: string | SimpleGetOptions | URL, cb: Callback): ClientRequest | never;
```
how this can be the `URL`-symbol?
In the readme (and in js) it looks more like plain POJO (or a string):
https://github.com/feross/simple-get#readme

Reply:
My mistake, I assumed it accepts the same types as get method from http, thanks